services/autoscaling/src/simcore_service_autoscaling/models.py

```diff
@@ -44,7 +44,7 @@ def __sub__(self, other: "Resources") -> "Resources":
 
 @dataclass(frozen=True)
 class EC2InstanceType:
-    name: str
+    name: InstanceTypeType
```

Comment:
OK this is the second time this morning. `RequestRequest` and not `TypeType`? 
Please a bit of imagination with names ! :-)
Aha this comes from `types_aiobotocore_ec2.literals`!! OMG! :-D

Reply:
yes. complain to Amazon...